dev-requirements.txt

```diff
@@ -16,6 +16,7 @@ Django>=1.6.5
 bottle>=0.12.3
 tornado>=4.0
 pyramid>=1.5.2
+webapp2==2.5.2
```

Comment:
We haven't been pinning to versions of frameworks so that Travis CI will always test against the latest released version. For consistency, we should do the same with webapp2.